azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -82,6 +84,32 @@ public ProjectManager(final AzkabanProjectLoader azkabanProjectLoader,
     loadProjectWhiteList();
   }
 
+  public boolean hasFlowTrigger(final Project project, final Flow flow)
+      throws IOException {
+    final String flowFileName = flow.getId() + ".flow";
+    final int latestFlowVersion = this.projectLoader.getLatestFlowVersion(project.getId(), flow
+        .getVersion(), flowFileName);
+    if (latestFlowVersion > 0) {
+      final File tempDir = com.google.common.io.Files.createTempDir();
+      final File flowFile;
+      try {
+        flowFile = this.projectLoader
+            .getUploadedFlowFile(project.getId(), project.getVersion(),
```

Comment:
It is possible that getUploadedFlowFile throws ProjectManagerException. Do you plan to catch it as well?

Reply:
will update, thanks